autoPyTorch/pipeline/components/preprocessing/tabular_preprocessing/scaling/StandardScaler.py

```diff
@@ -22,7 +22,7 @@ def __init__(self,
     def fit(self, X: Dict[str, Any], y: Any = None) -> BaseScaler:
 
         self.check_requirements(X, y)
-
+        # TODO: have checking for sparsity done in Dataset
```

Comment:
Can we put this in the gdoc / in an issue? Here we might forget it

Reply:
Actually it has been implemented already so I'll remove this and use the issparse coming from the dataset